.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+* [ ] document all added functions
+* [ ] try in sandbox /simulation/labnet
+* [ ] unit tests on the added/changed features
+  * [ ] make tests compile
+  * [ ] make tests pass 
+* [ ] add logs allowing easy debugging in case the changes caused problems
+* [ ] add comments and docs
```

Comment:
replace "add comments and docs" (it's already at the top) with:

if the API has changed, update the API specification